pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -105,13 +105,7 @@ class GatewayDataForm extends React.Component {
           component={Input}
         />
         <Message component="h4" content={sharedMessages.lorawanOptions} />
-        <FrequencyPlansSelect
-          horizontal
-          source="gs"
-          name="frequency_plan_id"
-          menuPlacement="top"
-          required
-        />
+        <GsFrequencyPlansSelect horizontal name="frequency_plan_id" menuPlacement="top" required />
```

Comment:
you can remove `horizontal`
```suggestion
        <GsFrequencyPlansSelect name="frequency_plan_id" menuPlacement="top" required />
```